examples/example.py

```diff
@@ -8,6 +8,8 @@ def main():
     site = PVSite(latitude=51.75, longitude=-1.25, capacity_kwp=1.25)
     
     ts = datetime.today() - timedelta(weeks=1)
+
+    # User has three options for the 'nwp_source': 'icon', 'gfs', or 'ukmo_seamless'.
```

Comment:
how comes its called `ukmo_seamless`?

Reply:
That is the model name in the docs https://open-meteo.com/en/docs/ukmo-api#hourly=

![Uploading Screenshot from 2024-09-02 13-54-25.png…]()

Reply:
Could you call it `ukmo` in here, and then add `ukmo_seamless` to the model params?

Reply:
> Could you call it `ukmo` in here, and then add `ukmo_seamless` to the model params?

Could you elaborate further?

Reply:
To try and keep this code nice and readable, could we change this to be`nwp_source='ukmo'` 

Would need to change data.py to use `ukmo` but in line 80 keep that as `ukmo_seamless`

Reply:
OKay, thanks for the feedback. I'll get right to it.